ui-tests/src/test/java/org/openmrs/reference/page/EditPatientRelationshipPage.java

```diff
@@ -0,0 +1,34 @@
+package org.openmrs.reference.page;
+
+import org.openmrs.uitestframework.page.Page;
+import org.openqa.selenium.By;
+import org.openqa.selenium.support.ui.Select;
+
+public class EditPatientRelationshipPage extends Page {
+	
+    private  HomePage homePage;
+    private static String name = "John";
```

Comment:
Its better to set the String value within the test implementation than within the page. This would mean creating a generic method that can allow one to set the patient name.

Reply:
Resolved thanks

Reply:
The point here is eliminate setting the String value within the page but rather creating a generic method that can set the name within the test implementation that excising the page functionality.